HHS/OPRE-OPS
# Refactor Azure Document Repository to route through the App Gateway

## What changed

Previously, the app uploaded files directly to the blob storage. As part of our security efforts, we're routing document upload through the app gateway instead. At the moment there are some issues with CORS that I do not expect to be present on the dev branch because dev is deployed to the same url as the dev app gateway. Part of this PR is to test this in dev

## Issue

[OPS-2362](https://github.com/HHS/OPRE-OPS/issues/2362)

## How to test

1. Navigate to the dev page and log in as an admin user
2. Add /upload-document to the URL
3. upload a document via the UI
4. verify there were no failed requests in the network tab to a URL that has UUID?se={rest of token goes here)
5. Verify in azure doc storage that your file was uploaded under its UUID as the name


## Definition of Done Checklist
